app/templates/serviceLearning/slcProposal.html

```diff
@@ -1,6 +1,6 @@
 {%set title ="SLC Proposal"%}
-<h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course Designation</h2>
-<p class="text-center fw-bold"> Please review the <a href="/serviceLearning/newProposal">service learning course guidelines</a> before completing this form </p>
+<h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
+<p class="text-center fw-bold"> Please review the <a href="/serviceLearning/editProposal/{{courseID}}">Service-Learning course guidelines</a> before completing this form </p>
```

Comment:
This breaks if they are creating a new proposal. There are two different cases to handle in this link.